### DISCUSSION TITLE
Touching right side of added Image,  enlargement layer will not display.

### DISCUSSION THREAD

**Discussion by Develeste** [singerdmx/flutter-quill#491] [General]
[My issue is about [Mobile] latest flutter version in stable channel on branch master.]

In the read-only mode, if I touch the right side of the added image,

the photo enlargement layer do not display.

But if I touch the left side, the image enlargement layer displays.

I think it should all work the same when touching all areas of the image.

This is because users cannot expect it to work like that.

This happens the same even when you run the example app.

**Comment by apoleo88** [comment]
I encountered the same issue.

Feel like it is a bug, not a discussion.
